vendor/vendor.json

```diff
@@ -0,0 +1,268 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "E2a5TaCbwqs7PwP1fvfWLhvgr38=",
+			"comment": "v2.2.0-rc.0-72-g106d918",
+			"path": "github.com/coreos/etcd/Godeps/_workspace/src/github.com/coreos/go-systemd/journal",
```

Comment:
Ah, it now seems someone changed the code in `Godeps`. Check this out, https://github.com/coreos/etcd/blob/106d918dd585a63ee0045740fe040daa44669a77/etcdserver/stats/stats.go.